tools/docker/iroha-all-in-one/healthcheck.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+from iroha import Iroha, IrohaCrypto, IrohaGrpc
+
+#setup of iroha client address
+IROHA_HOST_ADDR = os.getenv('IROHA_HOST_ADDR', '127.0.0.1')
+IROHA_PORT = os.getenv('IROHA_PORT', sys.argv[2]) 
+net = IrohaGrpc(f'{IROHA_HOST_ADDR}:{IROHA_PORT}')
+#setup admin account
+ADMIN_ACCOUNT_ID = os.getenv('ADMIN_ACCOUNT_ID', 'admin@test')
+ADMIN_PRIVATE_KEY = os.getenv(
+    'ADMIN_PRIVATE_KEY', 'f101537e319568c765b2cc89698325604991dca57b9716b58016b253506cab70')
+iroha_admin = Iroha(ADMIN_ACCOUNT_ID)
+user_account = 'admin@test'
+iroha_user = Iroha(user_account)
+
+#health check function
+def get_account(account:str):
+    query = iroha_user.query('GetAccount', account_id=account)
+    IrohaCrypto.sign_query(query, ADMIN_PRIVATE_KEY)
+    response = net.send_query(query)
+    #health check passes when response contains "admin@test"
+    if response.account_response.account.account_id == user_account:
+        print("Succss!")  
+        sys.exit(0)
+    #health check fails when response does not contain "admin@test"
+    else:
+        print("Connection success, but account does not exists!")
+        sys.exit(1)
+
+if __name__ == '__main__':
+    try:
+        get_account(sys.argv[1])  
+    #health check fails when connection with Iroha client cannot establish      
+    except Exception as e:   
+        print("Connection failure!")
+        sys.exit(1)
```

Comment:
I suggest to format all python files with: https://github.com/psf/black because python programmers are trying to be compatible with PEP programming style, the tool will reformat Your code.

Reply:
Done. Thank you! That is a great tool.